app/models/user.rb

```diff
@@ -80,7 +80,7 @@ class User < ApplicationRecord
   validates :stackoverflow_url,
             allow_blank: true,
             format:
-            /\A(http(s)?:\/\/)?(www.stackoverflow.com|stackoverflow.com|www.stackexchange.com|stackexchange.com)\/.*\Z/
+            /\A(http(s)?:\/\/)?(((www|pt|ru|es|ja).)?stackoverflow.com|(www.)?stackexchange.com)\/.*\Z/
```

Comment:
Can you add a few tests to make sure that your changes work or are valid?

Take a look here: https://github.com/thepracticaldev/dev.to/blob/master/spec/models/user_spec.rb#L185-L195